projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/configurator-group-menu.component.migration.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  CLICK_ON_ENTER,
+  CONFIGURATOR_GROUP_MENU_COMPONENT,
+  NAVIGATE_UP_ON_ENTER,
+  SPARTACUS_PRODUCT_CONFIGURATOR,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+//feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts
+export const CONFIGURATOR_GROUP_MENU_COMPONENT_MIGRATION: MethodPropertyDeprecation[] = [
+  {
+    class: CONFIGURATOR_GROUP_MENU_COMPONENT,
+    importPath: SPARTACUS_PRODUCT_CONFIGURATOR,
```

Comment:
No, this needs to be imported from /rulebased endpoint

Reply:
Done!